target/classes/META-INF/resources/default.css

```diff
@@ -582,7 +582,51 @@ hr {
 		padding-left: 40px;
 	}
 
+/*********************************************************************************/
```

Comment:
Can you include screenshots of any UI where large changes have been made (except for something trivial like the spelling mistake you corrected)? This makes it easy for others who are reviewing your code to spot issues with the design/UI which is really hard to do just by looking at the code alone.